### PR TITLE
parameter model

### PR DESCRIPTION
**Pull request by Henrilang**

### CODE REVIEW

File: app/models/parameter.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Parameter < ApplicationRecord
+  belongs_to :user_id
+
+  validates :origin, presence: true
+  validates :destination, presence: true
+  validates :earliest_start, :latest_finish, presence: true
```
Comment (Shakey343) [inline app/models/parameter.rb:6]: Set up a default if presence is false

Comment (Shakey343) [reply app/models/parameter.rb:6]: Presence should not be true for these

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :parameters, only: [:create, :show,]
```
Comment (Shakey343) [inline config/routes.rb:4]: need to delete ','

**Review by Shakey343 — CHANGES_REQUESTED** [review]

File: app/models/parameter.rb (hunk 1 above)
Comment (Henrilang) [reply app/models/parameter.rb:6]: done

File: config/routes.rb (hunk 2 above)
Comment (Henrilang) [reply config/routes.rb:4]: done

**Review by Shakey343 — APPROVED** [review]

**Pull request merged**